TramsDataApi/Gateways/TrustGateway.cs

```diff
@@ -0,0 +1,24 @@
+using System.Linq;
+using TramsDataApi.DatabaseModels;
+using TramsDataApi.Factories;
+using TramsDataApi.ResponseModels;
+
+namespace TramsDataApi.Gateways
+{
+    public class TrustGateway : ITrustGateway
+    {
+        private readonly TramsDbContext _dbContext;
+
+        public TrustGateway(TramsDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+        
+        public TrustResponse GetByUkprn(string ukprn)
+        {
+            var group = _dbContext.Group.FirstOrDefault(g => g.Ukprn == ukprn);
```

Comment:
I'd change the var name to be something a bit more human friendly, mirroring the schema in our variables is going to couple the code to that and make it harder to understand. This is a trust, lets call it that and ignore the fact the table is called group (because that is a GIAS detail anyway :( )